Assets/Prefabs/ProjectorAnimation.anim

```diff
@@ -0,0 +1,116 @@
+%YAML 1.1
```

Comment:
We have an animations folder for a reason

Reply:
I need to clean my room and transfer that organization in my life lol